docs/ramalama-serve.1.md

```diff
@@ -91,6 +94,42 @@ CONTAINER ID  IMAGE                             COMMAND               CREATED
 7bb35b97a0fe  quay.io/ramalama/ramalama:latest  llama-server --po...  3 minutes ago  Up 3 minutes  0.0.0.0:43869->8080/tcp  MyGraniteServer
 ```
 
+### Generate a kubernetes YAML file named tini
+```
+$ ramalama --nocontainer serve --name tini --generate kube tiny
+# Save the output of this file and use kubectl create -f to import
+# it into Kubernetes.
+#
+# Created with ramalama-0.0.17
+apiVersion: v1
+kind: Pod
```

Comment:
Come to think about it I think a `Deployment` will be more useful than a `Pod`

Reply:
Changed